lib/Transforms/Instrumentation/KleeInternalize.cc

```diff
@@ -239,13 +243,13 @@ namespace
 
     bool runOnModule (Module &M)
     {
+      m_module = &M;
       declareKleeFunctions(M);
       internalizeVariables(M);
 
       for (Function &F : M)
       {
-        if (shouldInternalize (F)) defineFunction (F);
```

Comment:
Yes. We need to assign a unique name per call. I added a few lines of code to remove the initial declaration in the new commit.